src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -162,7 +163,9 @@ let rec load_with_max_length :
         ~root_data:(Lazy.force genesis_root_data)
     in
     let%bind () = Persistent_root.reset_to_genesis_exn persistent_root in
-    continue (Persistent_frontier.create_instance_exn persistent_frontier)
+    continue
+      (Persistent_frontier.create_instance_exn persistent_frontier)
+      ~ignore_consensus_local_state:false
```

Comment:
Should this not be `true` here? We don't necessarily know that our local state will be in sync here.